compat/src/index.js

```diff
@@ -182,7 +181,7 @@ function cloneElement(element) {
  * @returns {boolean}
  */
 function isValidElement(element) {
-	return element && element.$$typeof===REACT_ELEMENT_TYPE;
+	return element!=null && element.$$typeof===REACT_ELEMENT_TYPE;
```

Comment:
Added the null check here to ensure that `isValidElement` always returns a `boolean`.